### PR TITLE
Reorganize the Azure infra scale set + add billing configuration section

### PR DESCRIPTION
**Pull request by mbenguig**

### CODE REVIEW

**Comment by mbenguig** [comment]
I followed the UI order.. Either i remove this comment, otherwise i undo these changes and only keep the billing configuration section. But to my point of view, the easiest way to build a node source (from a user POV) is to use the rm portal and refer to the doc.

File: src/docs/admin/references/NodeSourceInfrastructureReference.adoc [hunk 1]
```diff
@@ -782,11 +800,19 @@ You can opt to place this file in `$PROACTIVE_HOME/config/authentication/azure.c
...
-- *cleaningDelay:* Periodical cleaning delay, expressed in _seconds_. Every _X_ seconds the Resource Manager will remove from the Scale Set any VM that has been identified as idle. Default value is _60_. Note that setting a very short interval implies more frequent communication between the Resource Manager and Azure, which might impact on performance or responsiveness.
-- *externalStorageAccount:* Optional parameter. Use it if you want to define a specific location of your ProActive node.jar worker; leave blank otherwise. If provided, the format should be: _{storageAccount}:{sas_key_token}[:has_node.jar]_.
+- *nodePreCommand:* Optional parameter. This multi-line field can be used to provide specific commands to execute right before launching the ProActive's node.jar application. The main difference with the previous user custom script is that _nodePreCommand_ will be run as _the same_ user as node.jar. This field is thus useful for operations that require matching users, e.g. mounting a shared volume in Windows or creating non-root paths/files in Linux. Please note the OS-specific conditions for these commands:
```
Comment (mcompastie) [inline src/docs/admin/references/NodeSourceInfrastructureReference.adoc]: -`right before` sounds a bit oral. I would suggest just `before`
-  `Contrary to the previous user custom script, _nodePreCommand_ will be run as _the same_ user as node.jar`

File: src/docs/admin/references/NodeSourceInfrastructureReference.adoc [hunk 2]
```diff
@@ -782,11 +800,19 @@ You can opt to place this file in `$PROACTIVE_HOME/config/authentication/azure.c
...
+* *_Linux:_* The commands will be appended as `ExecStartPre` lines to a _.service_ file. Therefore, commands should include their full path, e.g. `mkdir` should be written as `/bin/mkdir` or `chown` as `/bin/chown`.
```
Comment (mcompastie) [inline src/docs/admin/references/NodeSourceInfrastructureReference.adoc]: - `The commands will be appended as `ExecStartPre` lines to a _.service_ file` => Overcomplexified here. I would just say it is not executed in shell and executable should be specified with their absolute path.

File: src/docs/admin/references/NodeSourceInfrastructureReference.adoc [hunk 3]
```diff
@@ -782,11 +800,19 @@ You can opt to place this file in `$PROACTIVE_HOME/config/authentication/azure.c
...
+* *_Windows:_* The commands will be included in a _.bat_ batch file and must therefore be DOS-compatible. As a guideline, here is an unofficial https://www.robvanderwoude.com/batchcommands.php[list of DOS commands] and a http://steve-jansen.github.io/guides/windows-batch-scripting/part-1-getting-started.html[scripting tutorial].
```
Comment (mcompastie) [inline src/docs/admin/references/NodeSourceInfrastructureReference.adoc]: The command will be executed in a the context of the cmd.exe interpreter

File: src/docs/admin/references/NodeSourceInfrastructureReference.adoc [hunk 4]
```diff
@@ -782,11 +800,19 @@ You can opt to place this file in `$PROACTIVE_HOME/config/authentication/azure.c
...
+- *jvmParameters:* Add JVM parameters to configure your ProActive node, use the format _-Dproperty=value_. The default values allow for a correct deployment of the node. You can add your own parameters but be aware that removing the current values might cause the deployment to fail. A non-exhaustive list of node-related JVM parameters can be found in your configuration file `$PROACTIVE_HOME/config/network/node.ini`
```
Comment (mcompastie) [inline src/docs/admin/references/NodeSourceInfrastructureReference.adoc]: - `your ProActive nodes` maybe

**Pull request merged**